pkg/trace/interpreter/util/span_metadata.go

```diff
@@ -0,0 +1,19 @@
+package util
+
+import "github.com/StackVista/stackstate-agent/pkg/trace/pb"
+
+// SpanMetadata contains the fields of the span meta that we are interested in
+type SpanMetadata struct {
```

Comment:
I would avoid have `util` packages, can this be part of the `SpanInterpreterEngine` ?

Reply:
Couldn't get rid of this one. Renamed the package to `model`, I need an "in between" package to avoid a import cycle

Reply:
ok